nuxeo-features/nuxeo-platform-userworkspace/nuxeo-platform-userworkspace-core/src/main/java/org/nuxeo/ecm/platform/userworkspace/core/service/AbstractUserWorkspaceImpl.java

```diff
@@ -475,7 +457,12 @@ protected UserWorkspaceServiceImplComponent getComponent() {
 
     protected abstract DocumentModel doCreateUserWorkspacesRoot(CoreSession unrestrictedSession, PathRef rootRef);
 
+    protected abstract DocumentModel initCreateUserWorkspaceRoot(CoreSession unrestrictedSession, DocumentModel doc);
```

Comment:
initCreateUserWorkspace**s**Root for consistency with doCreateUserWorkspacesRoot?

Reply:
+1